kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/ProxyDeployment.java

```diff
@@ -46,12 +48,12 @@ static String deploymentName(KafkaProxy primary) {
     protected Deployment desired(KafkaProxy primary,
                                  Context<KafkaProxy> context) {
         // @formatter:off
-        return new DeploymentBuilder()
+        return addStandardLabels(primary, new DeploymentBuilder()
                 .editOrNewMetadata()
                     .withName(deploymentName(primary))
                     .withNamespace(primary.getMetadata().getNamespace())
                     .addNewOwnerReferenceLike(ResourcesUtil.ownerReferenceTo(primary)).endOwnerReference()
-                    .withLabels(deploymentLabels())
+                    .addToLabels(APP_KROXY))
```

Comment:
Wouldn't it look nicer at call sites if the method was `StandardLabels.standardLabels()`, which returned a `Map`, so the callsite could be `.addToLabels(StandardLabels.standardLabels())`? That would avoid the awkward wrapping of the Fluent